src/test/store/useful-tabs.test.js

```diff
@@ -33,9 +33,6 @@ describe('getUsefulTabs', function () {
     const profile = getProfileWithMarkers(getNetworkMarkers());
```

Comment:
Could you add a small tests for this new behavior too? We already see that in some of the tests, we don't have the sample related tabs anymore since the profiles only include markers, but it could be nice to add a simple test that checks this new behavior only. 
The test name could be something like `does not show sample related tabs when there is no sample data in the profile` and it could test a profile like `const profile = getMarkerTableProfile()` or `getProfileWithMarkers(getNetworkMarkers());`